kdlib/source/kdlib.vcxproj

```diff
@@ -455,24 +455,34 @@
+    <Error Condition="!Exists('$(SolutionDir)\packages\boost.1.72.0.0\build\boost.targets')" Text="$([System.String]::Format('$(ErrorText)', '$(SolutionDir)\packages\boost.1.72.0.0\build\boost.targets'))" />
+    <Error Condition="!Exists('$(SolutionDir)\packages\boost_atomic-src.1.72.0.0\build\boost_atomic-src.targets')" Text="$([System.String]::Format('$(ErrorText)', '$(SolutionDir)\packages\boost_atomic-src.1.72.0.0\build\boost_atomic-src.targets'))" />
+    <Error Condition="!Exists('$(SolutionDir)\packages\boost_chrono-src.1.72.0.0\build\boost_chrono-src.targets')" Text="$([System.String]::Format('$(ErrorText)', '$(SolutionDir)\packages\boost_chrono-src.1.72.0.0\build\boost_chrono-src.targets'))" />
+    <Error Condition="!Exists('$(SolutionDir)\packages\boost_date_time-src.1.72.0.0\build\boost_date_time-src.targets')" Text="$([System.String]::Format('$(ErrorText)', '$(SolutionDir)\packages\boost_date_time-src.1.72.0.0\build\boost_date_time-src.targets'))" />
+    <Error Condition="!Exists('$(SolutionDir)\packages\boost_regex-src.1.72.0.0\build\boost_regex-src.targets')" Text="$([System.String]::Format('$(ErrorText)', '$(SolutionDir)\packages\boost_regex-src.1.72.0.0\build\boost_regex-src.targets'))" />
+    <Error Condition="!Exists('$(SolutionDir)\packages\boost_system-src.1.72.0.0\build\boost_system-src.targets')" Text="$([System.String]::Format('$(ErrorText)', '$(SolutionDir)\packages\boost_system-src.1.72.0.0\build\boost_system-src.targets'))" />
+    <Error Condition="!Exists('$(SolutionDir)\packages\boost_thread-src.1.72.0.0\build\boost_thread-src.targets')" Text="$([System.String]::Format('$(ErrorText)', '$(SolutionDir)\packages\boost_thread-src.1.72.0.0\build\boost_thread-src.targets'))" />
+    <Error Condition="!Exists('..\..\..\packages\boost.1.72.0.0\build\boost.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\..\packages\boost.1.72.0.0\build\boost.targets'))" />
+    <Error Condition="!Exists('..\..\..\packages\boost_chrono-src.1.72.0.0\build\boost_chrono-src.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\..\packages\boost_chrono-src.1.72.0.0\build\boost_chrono-src.targets'))" />
+    <Error Condition="!Exists('..\..\..\packages\boost_date_time-src.1.72.0.0\build\boost_date_time-src.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\..\packages\boost_date_time-src.1.72.0.0\build\boost_date_time-src.targets'))" />
+    <Error Condition="!Exists('..\..\..\packages\boost_regex-src.1.72.0.0\build\boost_regex-src.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\..\packages\boost_regex-src.1.72.0.0\build\boost_regex-src.targets'))" />
+    <Error Condition="!Exists('..\..\..\packages\boost_system-src.1.72.0.0\build\boost_system-src.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\..\packages\boost_system-src.1.72.0.0\build\boost_system-src.targets'))" />

```

Comment:
Similar with these "`..\..\..\`". If possible, I'd prefer these paths to be relative to the `SolutionDir` that the main pykd solution is in.